src/crackmes/hackthebox/find-the-easy-pass/find-the-validation-routine.md

```diff
@@ -0,0 +1,136 @@
+# Find the validation routine
+
+The "Wrong Password!" string is located at 0x00454200 in the CODE section.

```

Comment:
i think you should move the comment below up here on how to find that string

Reply:
I hope that's ok with last changes ?